src/main/java/io/knotx/junit5/wiremock/KnotxWiremockExtension.java

```diff
@@ -143,16 +154,31 @@ private WireMockServer setupWiremockServer(KnotxWiremock knotxWiremock, Store st
     server.start();
 
     port = server.port();
-    WireMock.configureFor("localhost", port);
+    getOrCreateWiremock(port);
     map.put(port, server);
 
     return server;
   }
 
+  private static WireMock getOrCreateWiremock(int port) {
+    if (wiremockMap.containsKey(port)) {
```

Comment:
I think that this if statement should be after `wiremockMapLock.lock()`,

Reply:
done